wire-ios-transport/Source/Authentication/ZMAccessTokenHandler.m

```diff
@@ -191,7 +191,6 @@ - (void)sendAccessTokenRequestWithURLSession:(ZMURLSession *)URLSession clientID
         return;
     }
 
-    ZMLogInfo(@"Requesting access token from cookie (existing token: %p).", self.accessToken);
```

Comment:
I would put it as available for debug build with WireLogger

Reply:
I assume, even WireLogger logs will be available on the Internal builds

Reply:
true, I haven't looked at the implementation, but end result should be log visible in console with Debug configuration (not on DataDog)